scripts/EN/Cybil

```diff
@@ -2317,19 +2317,19 @@ Area(0x55); // 0x2f58: 00001026
 CleanScreen(); // 0x2f5e: 00000064
 LoadBackground(0x142); // 0x2f62: 00001069
 DrawBackground(); // 0x2f68: 00000065
-ShowText((6050)"Inside the hospital it is dark, and it seems that if one does not walk carefully then they would stumble."); // 0x2f6c: 0000100c
+ShowText((6050)"Inside the hospital it is dark, W1and it seems that if one does not walk carefully then they would stumble."); // 0x2f6c: 0000100c
 HandleInput(); // 0x2f72: 00000010
-ShowText((6051)"Pictures hanging on the walls, withered plants on the receptionist's desk, and other furnishings can be faintly seen."); // 0x2f76: 0000100c
+ShowText((6051)"Pictures hanging on the walls, W1withered plants on the receptionist's desk, W1and other furnishings can be faintly seen."); // 0x2f76: 0000100c
 HandleInput(); // 0x2f7c: 00000010
-ShowText((6052)"¥"Is your... Is your daughter really in this place?¥" ¥"...I don't know, but... all we can do is trust that woman from before... We don't have any other clues.¥" Harry spoke not expecting a reply."); // 0x2f80: 0000100c
+ShowText((6052)"¥"Is your…¥nW2Is your daughter really in this place?¥"¥n¥nW2¥"…I don't know, but… W2all we can do is trust that woman from before…¥nW2We don't have any other clues.¥"¥nW2Harry spoke not expecting a reply."); // 0x2f80: 0000100c
 HandleInput(); // 0x2f86: 00000010
-ShowText((6053)"He slowly turned the knob of a nearby door. It appears to be locked. He lets out a long sigh."); // 0x2f8a: 0000100c
+ShowText((6053)"He slowly turned the knob of a nearby door.¥nW2But it won't open.¥nW2It appears to be locked.¥nW2He lets out a long sigh."); // 0x2f8a: 0000100c
```

Comment:
Adding missing second sentence, "But it won't open."

Technically there is no but in the Japanese, but it makes more sense to have it.